starter-code/.jshintrc

```diff
@@ -0,0 +1 @@
+{"esversion": 6}

```

Comment:
Instead of putting that in every file, you can create a file called .jshintrc in the root folder and, inside it write:
{ "esversion": 6 }